UKHomeOffice/nmsw-ui
# NMSW-324 Crew routes

# Ticket

NMSW-324

----

## AC

Page | URL | Link type | Link name | Link route | Next page
-- | -- | -- | -- | -- | --
Upload crew details | /report-voyage/upload-crew-details | primary submit button | Check for errors | /report-voyage/crew-details-uploaded | noErrors: Crew details uploaded
Upload crew details | /report-voyage/upload-crew-details | primary submit button | Check for errors | stay on page | Errors:
Crew details uploaded | /report-voyage/crew-details-uploaded | primary submit button | Save and continue | /report-voyage | Draft declaration tasklist

Prototype: /mvp-2/add-crew-details

----

## To test

- run tests, all should pass
- run app
- go to `/report-voyage/upload-crew-details`
- click `save and continue`
- > you should be shown the no errors found page


----

## Notes
